### PR TITLE
Adds additional vignette

### PR DESCRIPTION
**Pull request by stephhazlitt**
Additional vignette provides an example trend analysis workflow.

### CODE REVIEW

**Comment by stephhazlitt** [comment]
Ready for a code & usefulness review @ateucher, at your convenience.

File: DESCRIPTION [hunk 1]
```diff
@@ -29,6 +29,9 @@ Suggests:
     rgdal,
     rmarkdown,
     sp,
-    testthat
+    testthat,
+    gghighlight,
+    EnvStats,
+    tibble
```
Comment (ateucher) [inline DESCRIPTION]: I'm not sure if you need `tibble` here - as it's installed with dplyr (not that it hurts...)

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# bcgroundwater 1.1
+
+* Adds example workflow vignette
```
Comment (ateucher) [inline NEWS.md]: Singular 'Add' I think

File: inst/doc/bcgroundwater-example-workflow.Rmd [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteDepends{gghighlight, EnvStats, tibble}
+---
+
+```{r set-options, echo=FALSE, message=FALSE, warning=FALSE}
+knitr::opts_chunk$set(fig.width = 7, fig.height = 4, message=FALSE, warning=FALSE)
+```
+
+One example of how to generate a reproducible trend analysis on B.C. groundwater level data using the `bcgroundwater` R package.
```
Comment (ateucher) [inline inst/doc/bcgroundwater-example-workflow.Rmd]: How about something like 'This vignette illustrates one example of how to...'

File: inst/doc/bcgroundwater-example-workflow.Rmd [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+       y = "Depth Below Ground (metres)") 
+```
+
+## Seasonal Kendall Trend Test
+
+Ensure complete set of median monthly values for each year and implement a Seasonal Kendall trend test:
+
+```{r }
+# use years with all (12) months data
+full_monthly_data_comp_yrs <- full_monthly_data %>%
```
Comment (ateucher) [inline inst/doc/bcgroundwater-example-workflow.Rmd:146]: Not important, but here I think you can do:

```r
full_monthly_data_comp_yrs <- full_monthly_data %>% 
  group_by(Year) %>% 
  filter(n() == 12) %>% 
  ungroup()
```

**Review by ateucher — APPROVED** [review]
Looks great! Just a couple of very minor comments/nitpicks :)

**Comment by ateucher** [comment]
I guess the only other thing is if we want any discussion/comparison of the two methods. They both give similar (overall) results; Seasonal M-K has more power than using average annual values... etc

**Comment by stephhazlitt** [comment]
Thanks! I will incorporate the useful nitpicks 👍 I think my thought was more around here is some 'how to' code, some text around test choice would be useful. Happy to accept a PR 😉.

**Pull request merged**